modules/apps/commerce/commerce-product-type-virtual-web/src/main/resources/META-INF/resources/details.jspf

```diff
@@ -14,6 +14,7 @@
  */
 --%>
 
+details.jspf
```

Comment:
@jonmak08 this is likely what's causing CI SF errors :slightly_smiling_face:

Reply:
gotcha.  i'm still testing this to make sure it works, thats why I kept those in there.  I am surprised the "text" is causing the error, but It doesn't surprise me.  Thanks for the feedback!